src/app/submission/submission-shared/model/templates/default/default.template.v2.ts

```diff
@@ -0,0 +1,379 @@
+        icon: 'fa-link',
+        uniqueCols: true,
+        allowImport: true,
+        rowAsSection: true,
+        columnTypes: [
+          {
+            name: 'Link',
+            valueType: { name: 'idlink' },
+            display: 'required'
+          },
+          {
+            name: 'Description',
+            valueType: { name: 'text' },
+            display: 'desirable'
+          }
+        ]
+      },
+      {
+        name: 'File',
+        description:
+          'List and describe the data files associated with your study. It is possible to link entire directories, but note that for data consumers these will be served as zip files.',
+        icon: 'fa-file',
+        uniqueCols: true,
+        allowImport: true,
+        rowAsSection: true,
+        columnTypes: [
+          {
+            name: 'File',
+            valueType: { name: 'file' },
+            display: 'required'
+          },
+          {
+            name: 'Description',
+            valueType: { name: 'text' },
+            display: 'required'
+          },
+          {
+            name: 'Type',
+            valueType: { name: 'text' },
+            display: 'desirable'
+          }
+        ]
+      },
+      {
+        name: 'Publication',
+        description:
+          'Add the bibliography relevant to the study. Autofill is available when searching by ' +
+          '<a target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/">PubMed</a> identifier. ' +
+          'For other IDs, you may use ' +
+          '<a target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/pmctopmid/#converter">PubMed\'s converter</a>.',
+        icon: 'fa-book',
+        uniqueCols: true,
+        allowImport: false,
+        rowAsSection: true,
+        columnTypes: [
+          {
+            name: 'PMID',
+            valueType: { name: 'pubmedid' },
+            display: 'desirable'
+          },
+          {
+            name: 'Authors',
+            valueType: { name: 'text' },
+            display: 'required'
+          },
+          {
+            name: 'Title',
+            valueType: { name: 'text' },
+            display: 'required'
+          },
+          {
+            name: 'Year',
+            valueType: { name: 'text' },
+            display: 'required'
+          },
+          {
+            name: 'Volume',
+            valueType: { name: 'text' },
+            display: 'desirable'
+          },
+          {
+            name: 'Issue',
+            valueType: { name: 'text' },
+            display: 'desirable'
+          },
+          {
+            name: 'Type',
+            valueType: { name: 'text' },
+            display: 'optional'
+          },
+          {
+            name: 'Issn',
+            valueType: { name: 'text' },
+            display: 'optional'
+          },
+          {
+            name: 'DOI',
+            valueType: { name: 'text' },
+            display: 'optional'
+          }
+        ]
+      },
+      {
+        name: 'Funding',
```

Comment:
Is there any downside to just adding this to the v1 default template, instead of creating a v2 for a backwards-compatible change?

Reply:
I think this was decided in a weekly meeting to do so but let's confirm today just in case